public/wp-content/themes/glific/page-faq.php

```diff
@@ -8,24 +8,80 @@
 
 get_header();
 
-$faqs = get_field('faq_content');
-$page_title = get_field('faq_page_title');
+$page_title = get_field( 'faq_page_title' );
```

Comment:
For page titile , You can also use `get_the_title()` function.

Reply:
Thanks @mohitgusain, I was using a separate field altogether for the title as It was something different from what we want as the slug (URL) but that can be handled. Worked now.